### PR TITLE
adjust resource memory limits on neon-system-db from 256Mi to 512Mi

### PR DESCRIPTION
**Pull request by maddadder**

### CODE REVIEW

File: Lib/Neon.Kube.Setup/KubeSetup.Operations.cs [hunk 1]
```diff
@@ -2639,7 +2639,7 @@ private static async Task DeployOpenEbsWithcStor(ISetupController controller, No
                             },
                             PoolConfig = new V1CStorPoolConfig()
                             {
-                                DataRaidGroupType = DataRaidGroupType.Stripe,
```
Comment (marcusbooyah) [inline Lib/Neon.Kube.Setup/KubeSetup.Operations.cs:2642]: Jeff and I talked about this recently. The issue here is the k8s client not respecting the `EnumMember` properties. We decided that the best solution is to just use constant strings instead here.

**Review by marcusbooyah — CHANGES_REQUESTED** [review]

**Comment by marcusbooyah** [comment]
Can you also describe the symptom you were seeing with neon-system-db? We'd like to keep the memory footprint to a minimum, and from what I've seen, 256M seems to be enough for most small clusters.

**Comment by maddadder** [comment]
> Can you also describe the symptom you were seeing with neon-system-db? We'd like to keep the memory footprint to a minimum, and from what I've seen, 256M seems to be enough for most small clusters.

I don'k know why, but on my machine i am seeing the memory footprint at 330Mb

File: Test/Test.Neon.Kube/KubeHelper/Test_Json.cs [hunk 2]
```diff
@@ -49,7 +49,7 @@ public void SerializeEnum()
 
             serializerOptions.Converters.Add(new JsonStringEnumMemberConverter());
 
-            var t = DataRaidGroupType.stripe;
+            var t = HostingCapabilities.Pausable;
```
Comment (marcusbooyah) [inline Test/Test.Neon.Kube/KubeHelper/Test_Json.cs:52]: Is this right?

**Review by marcusbooyah — COMMENTED** [review]
Thanks - looks good! Is the change in the test correct though?

File: Test/Test.Neon.Kube/KubeHelper/Test_Json.cs (hunk 2 above)
Comment (maddadder) [reply Test/Test.Neon.Kube/KubeHelper/Test_Json.cs:52]: I changed the SerializeEnum() to test against an actual enum like the method name suggests. Now that DataRaidGroupType is no longer an enum, and changed SerializeEnum() to test a random enum I found in the source code. I wasn't sure how to run the test so I'm not sure if it works.

**Pull request merged**